### ISSUE TITLE
Angular 8 compatibility

### ISSUE THREAD

**Issue by paugp** [rxweb/rxweb#188]
# Describe the bug
I created an Angular 8 project and when I use propArray doent work and crash.
`TypeError: Class constructor ModelWithArray cannot be invoked without 'new'`
The same code (and @rxwe/reactive-form-validators version) in Angular 7 works fine.

# To Reproduce
Create an angular 8 project.
Create two model, one with @propArray(OtherModel)
Create a form that create the formGroup. (No need to have html form)
`this.formGroup = formBuilder.formGroup<ModelWithArray>(new ModelWithArray());`
When component load, crash.
`TypeError: Class constructor ModelWithArray cannot be invoked without 'new'`

# Package Version
@rxweb/reactive-form-validators@1.9.1

# Browser
Chrome, Firefox, Opera (latest version)

# Additional Info
Angular 8 project now uses **es2015** as default tsconfig target.
The same code in Angular 7 with **es5** works.

**Comment by sholkajadav11** [comment]
@paugp  Can you paste your model definition, so that I can guide you the same.

**Comment by paugp** [comment]
Its imposible to reproduce with stackblitz. On stackblitz you can't modify teh tsconfig.json file.
The problem is with es2015 target.
Download the stackblitz code and change the tsconfig to compile with es2015 target.
```
{
  "compilerOptions": {
  …
  "module": "esnext",
  "moduleResolution": "node",
  …
  "target": "es2015",
  …
}
```
https://blog.angular.io/version-8-of-angular-smaller-bundles-cli-apis-and-alignment-with-the-ecosystem-af0261112a27

**Comment by paugp** [comment]
![image](https://user-images.githubusercontent.com/1235882/61199071-d353a100-a6dc-11e9-841c-2f602ce5d0d5.png)

**Comment by ajayojha** [comment]
@paugp  Thanks for sharing the detailed information. I am considering this as a bug and this will be resolved in our next release.

**Comment by ajayojha** [comment]
The issue has been resolved in 1.9.2 release. Please update your package.
